apple-pie/styles/style.css

```diff
@@ -0,0 +1,93 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: "Calibri Light";
+}
+
+#header {
+    background-image: url("../images/apple-pie.jpg");
+    background-size: cover;
+    height: 100vh;
+    width: 100vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
A good approach with the "background-image", but just one thing you could have made this height a bit shorter. ;)